### DISCUSSION TITLE
Problema invio mail

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#755] [Richieste di aiuto]
Buonasera,purtroppo stiamo riscontrando il seguente problema:quando si cerca di inviare le credenziali via mail agli studenti/docenti o a qualsiasi persona presente nel sistema,il messaggio dell'invio riuscito esce ,ma poi quando si va nella propria posta si nota che non è arrivato niente ,nonostante abbiamo effettuato la configurazione Email di tipologia "Server locale tramite PHP" usando Altervista , anche qui ,provando ad inviare una mail di prova ,risulta sempre il messaggio che dice che il tutto è andato a buon fine,ma nel momento in cui si accede alla propria posta elettronica,non risulta niente .

**Comment by trinko** [comment]
La cosa più probabile è che dipenda da Altervista: mi risulta che esistano delle limitazioni sull'invio delle email per i servizi gratuiti. Non so dirti di più, dovresti chiedere al loro supporto conferma di esistanza di tali limitazioni.
